### DISCUSSION TITLE
download content in foreground

### DISCUSSION THREAD

**Discussion by BRIAN-JESHUA** [aldostools/webMAN-MOD#1018] [General]
Hello, is it possible to block the download in the background, thus forcing the download in the foreground using webman?
![Sin título](https://github.com/aldostools/webMAN-MOD/assets/65926757/4c690c7c-9bb2-4fbc-9224-a9bac1e5cf20)

**Comment by aldostools** [comment]
Hello @BRIAN-JESHUA 
The button for "download in background" is a default feature of download_plugin.

I think there must be some parameters to force download in foreground. 
☹️Unfortunately I don't know how to call that function using VSH api.

You could test if changing the file extension behaves different.
